scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
Assuming you **can** have multiple `after=` lines (does anyone know this?), could you add `After=postgresql` in addition to `after=network.target` so that we are depending on both the network and postgres being started?